packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -148,7 +148,8 @@ contract TaikoL1 is EssentialContract, ITaikoL1, TaikoEvents {
     /// @inheritdoc ITaikoL1
     function proveBlocks(
         uint64[] calldata _blockIds,
-        bytes[] calldata _inputArr
+        bytes[] calldata _inputArr,
+        bytes proof
```

Comment:
if `proof` is "", we can still call the verifyProof inside LibProving.sol so we support the current non-aggregation mode?